torch/csrc/jit/codegen/cuda/lower_unroll.cpp

```diff
@@ -184,8 +184,7 @@ void UnrollPass::handle(ForLoop* fl) {
     } // for expr
   } else { //  if(!within_unroll)
     // modify in place, so grab a copy of exprs first.
-    std::vector<Expr*> exprs(
-        fl->body().exprs().begin(), fl->body().exprs().end());
+    const std::vector<Expr*> exprs = fl->body().exprs();
```

Comment:
That's fair, not sure why I insist on using the range constructor, though I have been consistently.

Reply:
If I had to venture a guess, it's probably because std::vector<T> is not covariant, so in many cases you really have to use the range constructor. But fortunately, not in this case.